README.rst

```diff
@@ -516,26 +515,6 @@ To extract packages present on the resulting image run:
     "layers": [
       "2176639d844bbe1386912e1d9952cebdb8249923a16691025cf693963f8aec53"
     ],
-    "mercator": [
-      {
-        "digests": {
-          "manifest": "10460bb1fe6c167f6ef25f56cf940fab6fb40dd1"
-        },
-        "ecosystem": "Python-Dist",
-        "path": "/usr/lib/python3.6/site-packages/iniparse-0.4-py3.6.egg-info/PKG-INFO",
-        "result": {
-          "author": "Paramjit Oberoi",
-          "author-email": "param@cs.wisc.edu",
-          "classifier": "Development Status :: 5 - Production/Stable\nIntended Audience :: Developers\nLicense :: OSI Approved :: MIT License\nLicense :: OSI Approved :: Python Software Foundation License\nOperating System :: OS Independent\nProgramming Language :: Python\nProgramming Language :: Python :: 2Programming Language :: Python :: 2.6\nProgramming Language :: Python :: 2.7\nProgramming Language :: Python :: 3\nProgramming Language :: Python :: 3.3\nProgramming Language :: Python :: 3.4Topic :: Software Development :: Libraries :: Python Modules",
-          "description": "iniparse is an INI parser for  Python which is API compatible\nwith the standard library's ConfigParser, preserves structure of INI\nfiles (order of sections & options, indentation, comments, and blank\nlines are preserved when data is updated), and is more convenient to\nuse.",
-          "home-page": "http://code.google.com/p/iniparse/",
-          "license": "MIT",
-          "name": "iniparse",
-          "platform": "UNKNOWN",
-          "summary": "Accessing and Modifying INI files",
-          "version": "0.4"
-        }
-      },
```

Comment:
I think the `"mercator"` field goes until ~L810, could you remove the rest of it from this example?